ui/page/settingsStripeCard/view.jsx

```diff
@@ -181,18 +181,19 @@ class SettingsStripeCard extends React.Component<Props, State> {
               },
               'post'
             ).then((customerSetupResponse) => {
-              console.log(customerSetupResponse);
-
               clientSecret = customerSetupResponse.client_secret;
 
               // instantiate stripe elements
               setupStripe();
             });
+            // 500 error from the backend being down
           } else if (error === 'internal_apis_down') {
-            var displayString = 'There was an error from the server, please let support know';
+            const displayString = 'There was an error from the server, please let support know';
             doToast({ message: displayString, isError: true });
           } else {
-            console.log('Unseen before error');
+            // probably an error from stripe
+            const displayString = 'There was an error getting your card setup, please let support know';
```

Comment:
Just put these like
CARD_SETUP_ERROR = __('blah');
API_SERVER_ERROR = __('bleg'); at the top.